docs/getting_started.rst

```diff
@@ -17,8 +17,8 @@ constructing either a ``Cloudant`` or ``CouchDB`` client.  Since connecting to
 the Cloudant managed service provides extra end points as compared to a CouchDB 
 instance, we provide the two different client implementations in order to 
 connect to the desired database service.  Once the client is constructed, 
-you follow that up by connecting to the account, performing your tasks, and then 
-disconnecting from the account.
+you follow that up by connecting to the instance, performing your tasks, and
```

Comment:
Using instance here doesn't quite sound right, perhaps it should be `server` rather than `instance`?